README.md

```diff
@@ -38,7 +38,7 @@ $ brew install xhyve
 ```
 $ git clone https://github.com/ailispaw/boot2docker-xhyve
 $ cd boot2docker-xhyve
-$ make
+$ make    # You may be asked for your sudo password
```

Comment:
I don’t think that’s correct. You don’t need to type `sudo make`. You type `make` and it asks for your password.